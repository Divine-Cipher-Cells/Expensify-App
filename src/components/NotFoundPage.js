import React from "react";
import {Link} from "react-router-dom";
const NotFoundPage = (props) => (
    <div>
    404! ~ <Link to="/">Home</Link>
    </div>
);
export default NotFoundPage;