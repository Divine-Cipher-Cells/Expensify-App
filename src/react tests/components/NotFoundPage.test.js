import NotFoundPage from "../../components/NotFoundPage";
import React from "react";
import { shallow } from "enzyme";


test("not found page component",() => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
    });