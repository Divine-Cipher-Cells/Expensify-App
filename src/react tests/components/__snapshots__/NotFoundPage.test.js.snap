// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`not found page component 1`] = `
<div>
  404! ~ 
  <Link
    to="/"
  >
    Home
  </Link>
</div>
`;
